tests/run-make/crate-hash-rustc-version/Makefile

```diff
@@ -1,38 +0,0 @@
-# ignore-cross-compile
-include ../tools.mk
-
-# Ensure that crates compiled with different rustc versions cannot
-# be dynamically linked.
-
-FLAGS := -Cprefer-dynamic -Csymbol-mangling-version=v0
-UNAME := $(shell uname)
-ifeq ($(UNAME),Linux)
-  EXT=".so"
-  NM_CMD := nm -D
-endif
-ifeq ($(UNAME),Darwin)
-  EXT=".dylib"
-  NM_CMD := nm
-endif
-
-ifndef NM_CMD
-all:
-	exit 0
```

Comment:
The test is only supposed to run on linux or apple. Is there something like 
```
//@ only-linux
//@ only-apple 
```
that would work?
I think that ^ always ignores, although I haven't tested it.

Reply:
We could try ignore-windows

Reply:
I'll have a look later to i see if theres something like only-unix